kolibri/core/bookmarks/models.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import models
+from morango.models import UUIDField
+
+from kolibri.core.auth.models import FacilityUser
+
+
+class Bookmark(models.Model):
+
+    id = UUIDField(primary_key=True, default=uuid.uuid4)
+    content_id = models.CharField(max_length=50, null=False, blank=False)
+    contentnode_id = models.CharField(max_length=50, null=False, blank=False)
+    channel_id = models.CharField(max_length=50, null=False, blank=False)
+    facility_user = models.ForeignKey(FacilityUser, blank=False)
+
+    morango_model_name = "bookmark"
+
```

Comment:
Can use the BaseLogModel as a template for some of this, but it will be important to set the partition correctly in order to ensure single user syncing happens:

https://github.com/learningequality/kolibri/blob/develop/kolibri/core/logger/models.py#L73

Can use roughly the same partition function here, but no need to account for the anonymous case.